components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -32,6 +32,7 @@ class SHOCMacrophysics : public scream::AtmosphereProcess
   using Spack                = typename SHF::Spack;
   using IntSmallPack         = typename SHF::IntSmallPack;
   using Smask                = typename SHF::Smask;
+  using view_1d_int_const    = typename KT::template view_1d<const Int>;
```

Comment:
Do we want a `const` view of int's to use or a non-const?  I feel like if I understand the usage in the cpp it should be non-const since you later set it to a specific value.  Maybe you need both a `view_1d_int` and a `view_1d_const`.

Reply:
@singhbalwinder , I think this question still applies.